.github/workflows/android_build.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
     - main
+  pull_request_target:
+  # TODO(yannic): Remove when #26109 is merged and the required checks migrated
+  # to `pull_request_target`.
   pull_request:
```

Comment:
GitHub is not smart enough to see it's the same. The intent is to merge this PR, then ask a repo admin to change the required check (because they count as different checks), and then remove `pull_request`.

Alternatively, we can always remove the required check, land this with only `pull_request_target`, and then enable it again. Let me know what you prefer.

Reply:
ok I'll reassign a repo admin so they can coordinate this